gateway/modules/eventing/helpers.go

```diff
@@ -179,6 +179,23 @@ func (m *Module) generateProcessedEventRequest(eventID string) *model.UpdateRequ
 	}
 }
 
+func (m *Module) triggerDLQEvent(ctx context.Context, eventDoc *model.EventDocument) error {
+	req := model.QueueEventRequest{
+		Type: fmt.Sprintf("%s%s", utils.DLQEventTriggerSuffix, eventDoc.RuleName),
+		Payload: map[string]interface{}{
+			"id":   eventDoc.ID,
+			"type": eventDoc.Type,
+			"body": eventDoc.Payload,
+		},
+	}
+	token, err := m.adminMan.GetInternalAccessToken()
+	if err != nil {
+		return err
+	}
+	_, err = m.QueueEvent(ctx, m.project, token, &req)
```

Comment:
Can we skip the schema validation and security rules step? That could create a problem. Use an internal method